### ISSUE TITLE
In index html escape the descriptions 

### ISSUE THREAD

**Issue by rasca** [jquery/api.jquery.com#636]
When visiting http://api.jquery.com/ in jQuery.param() description it should say:

> Create a serialized representation of an array, a plain object, or a jQuery object suitable for use in a URL query string or Ajax request. In case a jQuery object is passed, it should contain \<input\> elements with name/value properties.

But instead of `<input>` it displays an input.

**Comment by AurelioDeRosa** [comment]
Thank you for catching this issue @rasca.

**Comment by rasca** [comment]
@AurelioDeRosa thanks for the quick fix! Maybe it can be set to escape the xml text when rendering so this won't happen again in the future?

**Comment by AurelioDeRosa** [comment]
@rasca I agree this would be useful (cc: @arthurvr @kswedberg). In the meantime I fixed the entry to be inline with the other entries that don't have codes in the summary.

**Comment by scottgonzalez** [comment]
If you're going to close this, then please open a new issue since the real problem still exists.
